cmd/outline-ss-server/main.go

```diff
@@ -227,21 +227,23 @@ func readConfig(filename string) (*Config, error) {
 
 func main() {
 	var flags struct {
-		ConfigFile    string
-		MetricsAddr   string
-		IPCountryDB   string
-		IPASNDB       string
-		natTimeout    time.Duration
-		replayHistory int
-		Verbose       bool
-		Version       bool
+		ConfigFile                     string
+		MetricsAddr                    string
+		IPCountryDB                    string
+		IPASNDB                        string
+		natTimeout                     time.Duration
+		replayHistory                  int
+		EnableIPKeyConnectivityMetrics bool
+		Verbose                        bool
+		Version                        bool
 	}
 	flag.StringVar(&flags.ConfigFile, "config", "", "Configuration filename")
 	flag.StringVar(&flags.MetricsAddr, "metrics", "", "Address for the Prometheus metrics")
 	flag.StringVar(&flags.IPCountryDB, "ip_country_db", "", "Path to the ip-to-country mmdb file")
 	flag.StringVar(&flags.IPASNDB, "ip_asn_db", "", "Path to the ip-to-ASN mmdb file")
 	flag.DurationVar(&flags.natTimeout, "udptimeout", defaultNatTimeout, "UDP tunnel timeout")
 	flag.IntVar(&flags.replayHistory, "replay_history", 0, "Replay buffer size (# of handshakes)")
+	flag.BoolVar(&flags.EnableIPKeyConnectivityMetrics, "enable_ip_key_connectivity_metrics", false, "Enables the collection and reporting of IPKey connectivity metrics")
```

Comment:
I think we can lean on the feature name more, and use something like `enalbe_tunnel_time_metrics`
I don't think "IP connectivity" expresses what we are measuring.